game/src/edit/traffic_signals/mod.rs

```diff
@@ -645,12 +645,12 @@ fn make_side_panel(
 
         let up_button = ctx
             .style()
-            .btn_primary_light_icon("system/assets/tools/arrow_up.svg")
+            .btn_primary_light_icon("../widgetry/icons/arrow_up.svg")
```

Comment:
are you OK with peeking across crate boundaries on the filesystem like this? 

It's admittedly a little weird, but:
1. this is a bin crate, not a lib crate, so it seems like less of an issue
2. it's nice to dedupe assets - not for filespace, but  just so I have less places to possibly update or refer to.

Reply:
I think the alternative would be to duplicate this files in each crate that uses them. Or possibly a new symlink in system/data/widgetry

Reply:
This works for me. If we ever split widgetry into a separate git repo / Cargo workspace, we'll have to revisit, but that isn't soon.